pkg/logger/logger.go

```diff
@@ -199,6 +201,7 @@ func (l *Logger) PrintAuthf(username string, req *http.Request, status AuthStatu
 		Client:        client,
 		Host:          requestutil.GetRequestHost(req),
 		Protocol:      req.Proto,
+		RequestID:     requestutil.GetRequestID(req),
```

Comment:
TODO: Add this to our default logging templates (OK for minor release). But add it as an important note.